src/odemis/driver/actuator.py

```diff
@@ -1419,11 +1421,23 @@ def __init__(self, name, role, children, caxes_map, axis_name, positions, fallba
             if not (len(pos) == 2 or type(pos) == str):
                 raise ValueError("Position %s needs to be of format list with exactly two entries. "
                                  "Got instead position %s." % (key, pos))
+        # TODO the default len for atol is 2
```

Comment:
Indeed, you probably could also change it to `None`, and replace that on the fly to `(0,) * len(children)`.